src/desktop/apps/home/client/index.coffee

```diff
@@ -113,8 +115,8 @@ module.exports.HomeView = class HomeView extends Backbone.View
 
 module.exports.init = ->
   user = CurrentUser.orNull()
-
-  new HomeView el: $('body')
+  splitTest('home_search_test').view()
+  new HomeView el: $('body'), testGroup: sd.HOME_SEARCH_TEST
```

Comment:
Yea, you can probably just skip `testGroup` and always read it out of `sd`.

Reply:
Yeah I wasn't sure if we should be accessing `sd` everywhere - in view objects, templates and client-side code. It's a little cleaner passing vars through to templates on the whole I think.